bin/cli

```diff
@@ -90,9 +90,27 @@ const getChosenKitDependency = () => {
     }
   }
   return defaultValue
-};
+}
+
+// do some heuristics to try and catch situations where a user has run
+// `npm start` (the wrong command) locally and warn them.
+function warnIfNpmStart (argv, env) {
+  if (
+    argv.command === 'start' && // if user ran serve script then assume they know what they want
+    env.NODE_ENV !== 'production' && // some hosting services set NODE_ENV
+    env.PORT === undefined && // some hosting services set PORT
+    env.PASSWORD === undefined // user should have set PASSWORD when setting up hosting
+  ) {
+    console.warn('Warning: It looks like you may have run the command `npm start` locally')
+    console.warn('if you are using the kit on a personal machine you should use `npm run dev`')
+    console.warn()
+    console.warn('If you are seeing this warning in production, it may indicate that your')
+    console.warn('hosting service needs further configuration.')
+    console.warn()
```

Comment:
@NoraGDS to review this

Reply:
whats the current logic to display this message?

Reply:
Suggested revision to content

```suggestion
    console.warn('Warning: It looks like you may have run the command `npm start` locally.')
    console.warn('try running `npm run dev`')
    console.warn()
    console.warn('If you see the above warning when trying to host your prototype online,')
    console.warn('it may be that your hosting service needs further configuration.')
    console.warn()
```

We still want more time to think about it though, ideally `npm run dev` would be the last thing the user sees, but swapping the order of the paragraphs around doesn't work either :/

Reply:
Suggested revision to content

```suggestion
    console.warn('Warning: It looks like you may have run the command `npm start` locally.')
    console.warn('try running `npm run dev`')
    console.warn()
    console.warn('If you see the above warning when trying to host your prototype online,')
    console.warn('it may be that your hosting service needs further configuration.')
    console.warn()
```

We still want more time to think about it though, ideally `npm run dev` would be the last thing the user sees, but swapping the order of the paragraphs around doesn't work either :/